tests/x11/rmt/rmt_feature.pm

```diff
@@ -44,6 +44,26 @@ sub run {
         assert_script_run("rmt-cli products disable $i") if ($i =~ /\d{4}/);
     }
 
+    # enable multiple products with product name at the same time in different way
+    assert_script_run("rmt-cli product enable SLES/12.3/x86_64 sle-module-live-patching/15/x86_64");
+    assert_script_run("rmt-cli product list | grep SLES/12.3/x86_64");
+    assert_script_run("rmt-cli product list | grep sle-module-live-patching/15/x86_64");
+    assert_script_run("rmt-cli product enable SLES/12.5/x86_64,sle-we/15.2/x86_64");
```

Comment:
Need use ',' or ' ' to separate different products?

Reply:
> Need use ',' or ' ' to separate different products?

No need to use 'or' to separate different products. This case tests enabling products with different commands, for example whether it works if it has single quote between products, use whitespace or comma between products.